packages/rbflabs-agoric-react-components/src/contexts/wallet/index.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Provider
+ */
+export { WalletProvider as AgoricWalletProvider } from './Wallet.provider';
+export type { WalletProviderProps as AgoricWalletProviderProps } from './Wallet.provider';
+
+/**
+ * Hooks
+ */
+export {
+  useWallet as useAgoricWallet,
+  useWalletNotifications as useAgoricWalletNotifications,
+} from './hooks';
```

Comment:
Is there any advantage in renaming these exports compared to using Agoric prefix directly in the names of exported items in the original modules?

Reply:
I believe it is unnecessary to repeat "Agoric" in every one of these functions, you know it is Agoric. Why not choose the shorter version?